pkg/upload/android-aab.go

```diff
@@ -48,14 +49,19 @@ func ProcessAndroidAab(apiKey string, applicationId string, buildUuid string, pa
 			}
 
 			log.Success(filepath.Base(path) + " expanded")
-
-			aabManifestPath = filepath.Join(tempDir, "base", "manifest", "AndroidManifest.xml")
-
 		} else {
 			return fmt.Errorf(path + " is not an AAB file")
 		}
 	}
 
+	if aabManifestPath == "" {
+		aabManifestPathExpected := filepath.Join(tempDir, "base", "manifest", "AndroidManifest.xml")
+		if utils.FileExists(aabManifestPath) {
+			aabManifestPath = aabManifestPathExpected
+			log.Info("Found app manifest at: " + aabManifestPath)
```

Comment:
AABs are a bit different as we'd expect to have a manifest.

I would suggest that we apply the same pattern as the other commands (i.e. only attempt to read it if the file exists and validate that required fields are present before upload). However I would replace this Info message with a Warn if it's not found (i.e. "app manifest not found in AAB") as the path you print here is going to be in the tmp dir and so not very useful.